bundles/binding/org.openhab.binding.swegonventilation/src/main/java/org/openhab/binding/swegonventilation/internal/ValueCache.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.swegonventilation.internal;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * Simple value cache.
+ *
+ * @author Pauli Anttila
+ */
```

Comment:
Please add an `@since 1.12.0` here.

Reply:
Done. I thought that since tag isn't used anymore.